src/xopen/__init__.py

```diff
@@ -688,6 +691,7 @@ def xopen(
     mode: str = "r",
     compresslevel: Optional[int] = None,
     threads: Optional[int] = None,
+    format: Optional[str] = None
```

Comment:
I believe it makes sense to turn this into a keyword-only argument (to force naming the argument)